service/src/main/java/bio/terra/workspace/service/privateresource/PrivateResourceCleanupService.java

```diff
@@ -157,7 +158,8 @@ private void runCleanupFlight(WorkspaceUserPair workspaceUserPair) {
             // workspace.
             .addParameter(WorkspaceFlightMapKeys.ROLE_TO_REMOVE, null);
     try {
-      userCleanupJob.submitAndWait(null);
+      // Skip the job access check as the "manager" role does not grant read access to a workspace.
```

Comment:
And why wouldn't they have read access to the workspace?

Reply:
`manager` is a Sam role that the WSM service account grants itself on all workspaces on creation in order to do "admin" actions like removing a user from the workspace without having end-user credentials handy. Only the WSM SA ever has this role, it's never given to users or other SAs. 

In this case, it's used to clean up users who have been removed from a workspace indirectly (e.g. they were removed from the group which gave them workspace membership), and there aren't user credentials available. The role is defined in Sam here: https://github.com/broadinstitute/sam/blob/develop/src/main/resources/reference.conf#L121. 

We don't grant that role read access to the workspace because it's only ever granted to WSM's SA and intended to be used for "admin" access. So far, I don't have a usecase where WSM's SA needs workspace read permission, and we want to keep the set of "admin" permissions we grant to WSM's SA as small as possible.